models/mapper.go

```diff
@@ -49,6 +53,14 @@ func RectifyKeywords(keywords []string, keywordsLimit int) []string {
 	return rectifiedKeywords[:keywordsLimit]
 }
 
+func ValidateTopics(topics []string) []string {
+	if topics == nil {
+		log.Warn(context.Background(), "warning : topics not present, i.e. null")
+		topics = []string{""}
```

Comment:
As mentioned before, this is not an empty array
https://go.dev/play/p/n3N-WRSJolC